src/core/server/legacy/integration_tests/logging.test.ts

```diff
@@ -87,7 +87,7 @@ describe('logging service', () => {
         const loggedString = getPlatformLogsFromMock(mockConsoleLog);
         expect(loggedString).toMatchInlineSnapshot(`
           Array [
-            "[xxxx-xx-xxTxx:xx:xx.xxx-xx:xx][INFO ][test-file] handled by NP",
+            "[xxxx-xx-xxTxx:xx:xx.xxx+xx:xx][INFO ][test-file] handled by NP",
```

Comment:
This change doesn't make any sense to me...

Reply:
Yea, I have no idea what could cause this... otoh both values are the exact same. (`+00:00` and `-00:00` are the same TZ...)

FWIW, this is the line where the date is generated in log entries:

https://github.com/elastic/kibana/blob/646732f96d4842bc1b66854f4a94c87bfc3959c0/src/core/server/logging/layouts/conversions/date.ts#L36-L37

So this goes down to `moment` itself...

Reply:
@pgayvallet we do use a mock setup with moment within the jest presets https://github.com/elastic/kibana/blob/master/packages/kbn-test/src/jest/setup/mocks.js

We have now splited the jest unit and the integration presets but we still use the same configuration. If the timezones are exactly the same I think we good to proceed here then, right?

Reply:
> we do use a mock setup with moment within the jest presets

Doesn't `setupFilesAfterEnv` in https://github.com/elastic/kibana/pull/105144/files#diff-8203fb667350d023ddcc06b414c86c1c59944923461e28803fbd7fc3ca1a00c0R17 override https://github.com/elastic/kibana/blob/565a72d5248cc21c6c9b3e36022320a86d0df943/packages/kbn-test/jest-preset.js#L59 ?
I'm not sure `jest` applies these mocks in the integration tests.

Reply:
UPD: I added `'<rootDir>/node_modules/@kbn/test/target_node/jest/setup/mocks.js',` to `setupFilesAfterEnv` section of jest `integration_tests/jest_preset`, re-compiled the package, revered the changes in the test file and tests passed locally.

Reply:
opened https://github.com/mistic/kibana/pull/14

Reply:
Really wish it was clearer what Jest is doing with this stuff, but the idea that the presets override the setup files list and the config file merges them isn't shocking to me. Thanks for digging in @mshustov!

Reply:
Awesome, thanks here @mshustov